content/docs/command-reference/get-url.md

```diff
@@ -34,23 +33,34 @@ directory will be placed inside.
 
 DVC supports several types of (local or) remote locations (protocols):
 
-| Type    | Description    | `url` format                               |
-| ------- | -------------- | ------------------------------------------ |
-| `local` | Local path     | `/path/to/local/data`                      |
-| `s3`    | Amazon S3      | `s3://mybucket/data`                       |
-| `gs`    | Google Storage | `gs://mybucket/data`                       |
-| `ssh`   | SSH server     | `ssh://user@example.com:/path/to/data`     |
-| `hdfs`  | HDFS to file\* | `hdfs://user@example.com/path/to/data.csv` |
-| `http`  | HTTP to file\* | `https://example.com/path/to/data.csv`     |
+| Type     | Description                  | `url` format example                                   |
+| -------- | ---------------------------- | ------------------------------------------------------ |
+| `s3`     | Amazon S3                    | `s3://bucket/key/to/data`                              |
+| `azure`  | Microsoft Azure Blob Storage | `azure://container/path/to/data`                       |
+| `gdrive` | Google Drive                 | `gdrive://<folder-id>/data`                            |
+| `gs`     | Google Cloud Storage         | `gs://bucket/path/to/data`                             |
+| `ssh`    | SSH server                   | `ssh://user@example.com:/path/to/data`                 |
+| `hdfs`   | HDFS to file\*               | `hdfs://user@example.com/path/to/data.csv`             |
+| `http`   | HTTP to file\*               | `https://example.com/path/to/data.csv`                 |
+| `webdav` | WebDav to file\*             | `webdavs://example.com/public.php/webdav/path/to/data` |
+| `local`  | Local path                   | `/path/to/local/data`                                  |
+| `remote` | Remote path\*                | `remote://remote-name/path/to/data`                    |
```

Comment:
Is this list correct? does `get-url` support `remote://` for example? Maybe there was a reason it wasn't included before. Cc @efiop @shcheklein thanks

Reply:
Oh and should OSS be included too? Or maybe just mentioned as part of S3

Reply:
I guess we are overcomplicating this a bit ... what was the problem with all previous path URLs we had?

Reply:
With these particular lists the main problem was that they were incomplete (still missing OSS).

Reply:
No, get-url doesn't support remote://

Reply:
Ah yeah that makes sense dah me 🤦 will remove it 🙂